core/esmf-aspect-meta-model-python/README.md

```diff
@@ -1,84 +1,141 @@
 
-A [GitHub action called 'Release'](https://github.com/eclipse-esmf/esmf-sdk-py-aspect-model-loader/actions/workflows/tagged_release.yml)
-has been set up for the `esmf-aspect-model-loader`. This action checks the code quality by running tests, the [static type checker MyPy](https://github.com/python/mypy) and
-the [code formatter 'Black'](https://github.com/psf/black).
+Poetry provides convenient functionality for working with dependencies in the project. 
+To automatically download and install all the necessary libraries, just run one command
+```console
+poetry install
+```
 
-## Set Up SAMM Aspect Meta Model for development
+### Download SAMM files
 
-In order to download the SAMM sources, it is required to run `poetry install` once in the `esmf-aspect-model-loader`
-module. There are two possibilities to download the SAMM files and extract the Turtle sources for the Meta Model.
+There are two possibilities to download the SAMM files and extract the Turtle sources for the Meta Model: 
+SAMM release or SAMM branch
 
-### Possibility 1 (downloading a release)
+#### Download SAMM release
 
-The `download_samm_release` script may be executed with
+This script downloads a release JAR-file from GitHub, extracts them for further usage in the Aspect Model Loader:
 
-```
+To run script, execute the next command.
+```console
 poetry run download-samm-release
-```  
+```
+The version of the SAMM release is specified in the python script.
+
+Link to all Releases: [SAMM Releases](https://github.com/eclipse-esmf/esmf-semantic-aspect-meta-model/releases)
 
-It downloads a release JAR-file from GitHub and extracts the SAMM Files.
-The version is specified in the python script.
+#### Download SAMM branch
 
-Link to all Releases: https://github.com/eclipse-esmf/esmf-semantic-aspect-meta-model/releases
+The script uses the GitHub API and downloads the files from the `main` GitHub branch. 
 
-### Possibility 2 (downloading from the repository)
+If the script is run in a pipeline, it uses a GitHub token to authorize. If the script is run locally, 
+the API is called without a token. This may cause problems because unauthorized API calls are limited.
 
-It may happen that there is no .jar file that is up to date with the changes of the SAMM.
-This script is an alternative to the `download_samm_release.py` and extracts the files from the repository
-directly instead of using the newest release.
+Run the next command to download and start working with the Aspect Model Loader.
+```console
+poetry run download-samm-branch
+```
+Link to all branches: [SAMM Releases](https://github.com/eclipse-esmf/esmf-semantic-aspect-meta-model/branches)
 
-The script uses the GitHub API and downloads the files from the `main` branch. If the script is run in a
-pipeline, it uses a GitHub token to authorize. If the script is run locally, the API is called without a token.
-This may cause problems because unauthorized API calls are limited.
+### Download SAMM CLI
```

Comment:
This section I don't understand, shouldn't this work transparently? I.e., users should not be required to know about this at all, it should "just work". samm-cli should be downloaded automatically "under the hood" when it's not present yet.